### PR TITLE
[DOC] Update changelog

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,30 +28,66 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 ## New features
 
-#### Build system
+#### Alphabet
+  * `cigar` can now be assigned from `std::string_views` ([\#2966](https://github.com/seqan/seqan3/pull/2966)).
```
Comment (smehringer) [inline CHANGELOG.md:32]:
```suggestion
  * `seqan3::cigar` can now be assigned from `std::string_views` ([\#2966](https://github.com/seqan/seqan3/pull/2966)).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -28,30 +28,66 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
...
+  * Added `seqan3::list_traits::repeat` ([\#2899](https://github.com/seqan/seqan3/pull/2899)).
 
 ## Notable Bug-fixes
 
+#### Core
+  * Added missing implementations for AVX512 ([\#2920](https://github.com/seqan/seqan3/pull/2920) and
+    [\#2926](https://github.com/seqan/seqan3/pull/2926)).
+
 #### IO
+  * Resolved an issue parsing FASTA files containing sequence IDs that start with `>`
```
Comment (smehringer) [inline CHANGELOG.md:52]: which issue? This is really abstract

File: CHANGELOG.md [hunk 3]
```diff
@@ -28,30 +28,66 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
...
+    ([\#2869](https://github.com/seqan/seqan3/pull/2869)).
 
-* Fixed an issue parsing FASTA files containing sequence IDs that start with `>`
- ([\#2869](https://github.com/seqan/seqan3/pull/2869)).
+#### Search
+  * Relaxed `kmer_hash_view::iterator` difference requirement ([\#2931](https://github.com/seqan/seqan3/pull/2931)).
+  * Improved performance of `seqan3::counting_vector::operator+=` ([\#2930](https://github.com/seqan/seqan3/pull/2930)).
```
Comment (smehringer) [inline CHANGELOG.md:57]: isn't this rather a feature?

File: CHANGELOG.md [hunk 4]
```diff
@@ -66,6 +102,20 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
   * Replaced `seqan3::views::to` (implemented via range-v3) with `seqan3::ranges::to` (implemented in SeqAn3).
     `seqan3::ranges::to` provides a subset of C++23's `std::ranges::to` and will be replaced with the STL-equivalent
     in a future version ([\#2969](https://github.com/seqan/seqan3/pull/2969)).
```
Comment (smehringer) [inline CHANGELOG.md:104]:
```suggestion
    in a future version ([\#2969](https://github.com/seqan/seqan3/pull/2969)).
  * Replaced the implementation of `seqan3::views::chunk`. It is now implemented in SeqAn3 and does not use 
    the range-v3 implementation anymore. ([\#2975](https://github.com/seqan/seqan3/pull/2975))
```

**Review by smehringer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -28,30 +28,68 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
...
 ## New features
 
-#### Build system
+#### Alphabet
+  * `seqan3::cigar` can now be assigned from `std::string_view` ([\#2966](https://github.com/seqan/seqan3/pull/2966)).
+  * Added `seqan3::views::char_strictly_to`. Behaves like `seqan3::views::char_to`, but throws on invalid
+    input ([\#2898](https://github.com/seqan/seqan3/pull/2898)).
+
+#### I/O
+ * Added an option to ignore blanks before IDs when reading FASTA-files
```
Comment (smehringer) [inline CHANGELOG.md:37]:
```suggestion
 * Added `seqan3::sequence_file_option::fasta_ignore_blanks_before_id` to ignore blanks before IDs when reading FASTA-files (e.g. `>       some_id` will store only `"some_id"` as id.)
```

**Review by smehringer — CHANGES_REQUESTED** [review]

**Review by smehringer — APPROVED** [review]

**Review by SGSSGene — APPROVED** [review]

**Pull request merged**